fmc/README.md

```diff
@@ -139,10 +139,11 @@ fields may not be changed or removed). Table revisions with different Major Vers
 | fuse_log_addr         | 4            | ROM        | Fuse Log Address.                                                                                        |
 | rt_dice_pub_key       | 96           | FMC        | RT Alias DICE Public Key.                                                                                |
 | rt_dice_sign          | 96           | FMC        | RT Alias DICE signature.                                                                                 |
+| ldevid_dice_sign      | 96           | FMC        | LDevID DICE signature.                                                                                 |
 | idev_dice_pub_key     | 96           | ROM        | Initial Device ID Public Key.                                                                            |
 | rom_info_addr         | 4            | ROM        | Address of ROMInfo struct describing the ROM digest and git commit.                                      |
 | rtalias_tbs_size      | 2            | FMC        | RT Alias TBS Size.                                                                                      |
-| reserved              | 128          |            | Reserved for future use.                                                                                 |
+| reserved              | 30           |            | Reserved for future use.                                                                                 |
 
 *FHT is currently defined to be 512 bytes in length.*
```

Comment:
New fields also need to be described in the section directly below

Reply:
Done.